### DISCUSSION TITLE
STM32F4 hangup issue

### DISCUSSION THREAD

**Discussion by Admsher** [stm32-rs/stm32f4xx-hal#820] [Q&A] [answered]
I am trying to run a loop function which receives continuous CAN data from motor , but when i start the motor, the program seems to hang up , is there any particular thing i should be looking for , the code is similar to this : [stm32f4xx-hal/examples/can-send.rs at master · stm32-rs/stm32f4xx-hal (github.com)](https://github.com/stm32-rs/stm32f4xx-hal/blob/master/examples/can-send.rs)
with some calculations in between inside the loop. Any help is appreciated.

**Comment by burrbull** [comment, marked as answer]
I think https://github.com/stm32-rs/bxcan should be better place to ask CAN related issues

**Reply by burrbull** [reply]
It also makes sense to ask in matrix messenger channel https://matrix.to/#/#rust-embedded:matrix.org

**Reply by Admsher** [reply]
Yes will surely see ask there as well, it mp seems like a hardware issue but just wanted to be sure I am not doing anything iny code
